templates/zerver/apps.html

```diff
@@ -86,10 +86,17 @@ <h3 class="center">{% trans %}Ok, I take it back, I'm sorry, please don't go.{%
                             <div class="wrapper">
                                 <h2>MacOS</h2>
                                 <div class="instructions">
-                                    <p class="center">You love your Mac. And you love Zulip. So what could be better than a Zulip app for Mac? Enjoy notifications for messages and PMs in your dock whether you're in Sublime, emacs, or Photoshop.</p>
+                                    <p class="center">
+                                        {% trans %}You love your Mac. And you
+                                        love Zulip. So what could be better
+                                        than a Zulip app for Mac? Enjoy
+                                        notifications for messages and PMs in
+                                        your dock whether you're in Sublime,
+                                        emacs, or Photoshop.{% endtrans %}
+                                    </p>
```

Comment:
I agree with the newlines being an absolute pain. That's why I tried to address it in https://github.com/zulip/zulip/commit/e710110a9ea94716b4eeca396e6f8fb4c62c02d3, but I recently found out that it doesn't seem to be working as expected.

My opinion about this is that we have to change the way `makemessages` processes the translation blocks, not how we present the translation strings. That way, we change the screwdriver, not all the screws around.

Thoughts?